scripts/start.sh

```diff
@@ -24,10 +24,6 @@ function sleep_until_mysql_is_ready() {
 }
 
 function setup_mysql() {
-  # Start mysqld
```

Comment:
Maybe users would confuse why `setup_mysql function` populated dataset instead of setting up a MySQL service.

How do you think adding a new function `setup_mysql_with_example_dataset`?

``` bash
function setup_mysql_with_example_dataset() {
    setup_mysql()
    populate_example_dataset()
}
```

Reply:
This change is trying to add "starting mysql" in populate_example_dataset and is not changing existing setup_mysql behavior. I suspect that moving the populate_example_dataset out of setup_mysql will break some existing tests.

Reply:
offline talked with @Yancey1989  we'll keep the populate_example_dataset as it was (only populating data) and add a new function to cover the function of starting local mysql data base and populating data.